app/createpage.js

```diff
@@ -1,4 +1,8 @@
 //Load common code that includes config, then load the app logic for this page.
 requirejs(['./common'], function (common) {
-    requirejs(['./app/create', '../lib/include']);
+    requirejs(['./app/create', '../lib/include'], callMe);
 });
+
+function callMe($, canvas, sub) {
```

Comment:
I don't know what this is supposed to do, but I can hardly imagine a case where it is a good function name.